lib/pdffield.js

```diff
@@ -220,6 +220,9 @@ let PDFField = (function PDFFieldClosure() {
         anData.w -= 0.5; //adjust combobox width
         anData.PL = {V: [], D: []};
         _.each(field.value, function(ele, idx) {
+            if (!Array.isArray(ele)) {
+                ele = [ele, ele];
```

Comment:
don't mutate the iterative object, change it to:
if (!Array.isArray(ele)) {
  anData.PL.D.push([ele, ele]);
}
else {
  anData.PL.D.push(ele[0]);
  anData.PL.V.push(ele[1]);
}
}

Reply:
Reassigning `ele` to a new array doesn't mutate anything here, but I'll change it so it's more obvious that this doesn't affect `field.value`.